src/Game.js

```diff
@@ -13,10 +14,12 @@ export default class Game {
       charges: new Group(),
     };
 
-    this.puck = new Puck(100, 100);
+    this.puck = new Puck(100, 300);
```

Comment:
Extract Puck starting position to a constant, consider calling reset() function on Game start instead of repeating the setup in 2 places.